vnext/Microsoft.ReactNative/Fabric/Composition/ImageComponentView.cpp

```diff
@@ -223,16 +240,21 @@ void ImageComponentView::ensureDrawingSurface() noexcept {
       case facebook::react::ImageResizeMode::Contain:
         surfaceBrush.Stretch(winrt::Microsoft::ReactNative::Composition::CompositionStretch::Uniform);
         break;
-      case facebook::react::ImageResizeMode::Center:
-        surfaceBrush.Stretch(winrt::Microsoft::ReactNative::Composition::CompositionStretch::None);
-        break;
       case facebook::react::ImageResizeMode::Repeat:
-        surfaceBrush.Stretch(winrt::Microsoft::ReactNative::Composition::CompositionStretch::UniformToFill);
-        // TODO - Hook up repeat
+        surfaceBrush.HorizontalAlignmentRatio(0.0f);
```

Comment:
Is there a reason here why we fallthrough instead of break?

Also, it looks like the ReactImageBrushCode sets the Horizontal and Vertical Alignement Ratios to values to 0.5f for other resize modes, is that not necessary here?